kubernetes-model-generator/kubernetes-model-core/src/test/java/io/fabric8/kubernetes/api/model/QuantityTest.java

```diff
@@ -117,13 +114,13 @@ public void testExponents() {
   @Test
   @DisplayName("Test equals operation")
   public void testEquality() {
-    assertTrue(new Quantity(".5Mi").equals( new Quantity("512Ki")));
-    assertTrue(new Quantity("1Gi").equals(new Quantity("1024Mi")));
-    assertFalse(new Quantity("1E").equals(null));
-    assertFalse(((Object)new Quantity("2P")).equals((Object)"2P"));
+    assertEquals(new Quantity(".5Mi"), new Quantity("512Ki"));
+    assertEquals(new Quantity("1Gi"), new Quantity("1024Mi"));
+    assertNotEquals(null, new Quantity("1E"));
```

Comment:
```suggestion
    assertNotNull(new Quantity("1E"));
```

Reply:
@manusa in this line, an object is newly created, and the code is trying to assert whether it's not null. The assertion seems to be redundant and can be removed itself. Let me know if that seems okay.

Reply:
Yes, that's exactly the way to go.

Reply:
This is done.